reader-osm/src/test/java/com/graphhopper/GraphHopperIT.java

```diff
@@ -1370,7 +1374,7 @@ public void testEncoderWithTurnCostSupport_stillAllows_nodeBasedRouting() {
         GHRequest req = new GHRequest(p, q);
         req.setVehicle("foot");
         GHResponse rsp = tmpHopper.route(req);
-        assertEquals("there should not be an error, but was: " + rsp.getErrors(), 0, rsp.getErrors().size());
+        assertEquals(0, rsp.getErrors().size(), "there should not be an error, but was: " + rsp.getErrors());
```

Comment:
The message shown when an assertion fails is now the last parrameter in the `assertXYZ` methods...